src/modules/dashboard/components/CreateColonyWizard/StepTokenChoice.tsx

```diff
@@ -57,7 +57,7 @@ const MSG = defineMessages({
   },
   selectTokenSubtitle: {
     id: 'dashboard.CreateColonyWizard.StepTokenChoice.existingTokenSubtitle',
-    defaultMessage: 'For example: DAI, EOS, SNT, etc',
+    defaultMessage: 'Add in the list of examples: UNI, SUSHI, & AAVE',
```

Comment:
I think Commodore meant that you should add UNI, SUSHI, and AAVE to the previous message. So the message would be something like this: `For example: DAI, EOS, SNT, UNI, SUSHI, AAVE, etc`